thirdparty/cmdconfig/commands/cmdeval/cmdeval.go

```diff
@@ -293,3 +297,29 @@ func (r *EvalFnRunner) preRunE(c *cobra.Command, args []string) error {
 	// don't consider args for the function
 	return nil
 }
+
+func (r *EvalFnRunner) postRun(_ *cobra.Command, args []string) {
+	if len(args) > 0 && args[0] == "-" {
+		return
+	}
+	path := "."
+	if len(args) > 0 {
+		path = args[0]
+	}
+	inout := &kio.LocalPackageReadWriter{
+		PackagePath:    path,
+		MatchFilesGlob: append(kio.DefaultMatch, v1alpha2.KptFileName),
+	}
+	f := &filters.FormatFilter{
+		UseSchema: true,
+	}
+	err := kio.Pipeline{
+		Inputs:  []kio.Reader{inout},
+		Filters: []kio.Filter{f},
+		Outputs: []kio.Writer{inout},
+	}.Execute()
+	if err != nil {
+		// do not throw error if formatting fails
+		return
+	}
```

Comment:
Looks like we can have a method can be moved under `pkgutils` or `pkg` and can be invoked from the postRuns of both `eval` and `render`

Reply:
@droot Updated.